src/app/model/schema.ts

```diff
@@ -15,6 +15,7 @@ export class Resource {
   id!: string;
   name!: string;
   cost!: number;
+  farm: string = "";
```

Comment:
Just the name of a farm? Do we anticipate to keep more information about farms in the future? Maybe preemptive naming of `farmName`?

Reply:
name should be displayName
Same with josh, what is farm in this context ? Can we generalize it

Reply:
resourceProvider ? or resourceProviderName ?

I think id prefer the former. Lets not over think it YAGNI etc... lets deal with generalization later.

Reply:
In figma the food boxes had a farm name associated with it https://www.figma.com/file/GLg0zPNnwMEGIb9TytCrxQ/Resilience-App?node-id=3527%3A6595

I can just call resourceProvider?